modules/core/src/main/java/org/apache/ignite/internal/streamer/StreamerSubscriber.java

```diff
@@ -223,13 +225,7 @@ private void close(@Nullable Throwable throwable) {
 
             var futs = pendingRequests.values().toArray(new CompletableFuture[0]);
 
-            CompletableFuture.allOf(futs).whenComplete((res, err) -> {
-                if (err != null) {
-                    completionFut.completeExceptionally(err);
-                } else {
-                    completionFut.complete(null);
-                }
-            });
+            CompletableFuture.allOf(futs).whenComplete(copyStateTo(completionFut));
```

Comment:
Can use static import for `CompletableFuture#allOf`?

Reply:
This is an unrelated change overall, I don't want to add even more

Reply:
Then, in theory, adding `copyStateTo` also does not apply to changes, since the ticket is about fixing a problem in the rebalance.

Reply:
that's my point exactly, I'm kinda doing a prohibited thing already, I don't want to make my case even worse)

Reply:
If you got in, then it’s better to do it, and not half-measure. 
You can do it on a separate ticket (trollface)

Reply:
But what problem does a static import solve? I think it looks ok both ways